docs/content/get-started/openshift-platform/supported-platforms.md

```diff
@@ -11,6 +11,4 @@ Before deploying NGINX Service Mesh in OpenShift, see the [Considerations]({{< r
 
 ## Supported Platforms
 
-NGINX Service Mesh has been tested to run on OpenShift version 4.8.
-
-Any self-managed RedHat OpenShift environment running with the above version can be used with NGINX Service Mesh. Externally managed environments such as Azure Red Hat OpenShift and Red Hat OpenShift Service on AWS may work, though they are not officially supported.
+Any self-managed RedHat OpenShift environment running with the versions listed in the [Technical Specifications]({{< ref "/about/tech-specs.md#supported-versions" >}}) can be used with NGINX Service Mesh. Externally managed environments such as Azure Red Hat OpenShift and Red Hat OpenShift Service on AWS may work, though they are not officially supported.
```

Comment:
Not _supported_ or not tested?

Reply:
We do not test those platforms. The "supported" terminology was pre-existing to my changes here. I don't know what the decision behind that was.

Reply:
Lets use "validated" in this case.  Please.

Reply:
Updated.